package.json

```diff
@@ -35,12 +35,14 @@
     }
   },
   "scripts": {
+    "a11y": "node scripts/a11y/index.js",
     "bootstrap": "lerna bootstrap",
     "bootstrap:hoist": "lerna bootstrap --hoist --nohoist=moment --nohoist=one-cerner-style-icons",
     "clean:all": "lerna clean --yes && node scripts/clean-modules/index.js",
     "clean:install": "npm run clean:all && npm install",
     "clean:obsolete-snapshots": "npm test -- -u",
     "compile": "lerna run compile",
+    "compiledRouteConfig": "babel dev-site-config/build/routeConfig.js --out-file dev-site-config/build/compiledRouteConfig.js",
```

Comment:
This change shouldn't be included in this PR. It looks like this PR got mixed with work being done for terra-ui.